.travis.yml

```diff
@@ -39,6 +39,18 @@ matrix:
       - pytest test --cov;
     after_success:
       - bash <(curl -s https://codecov.io/bash)
+  - name: "Pytest python 3.6 with skggm + scikit-learn 0.20.3"
```

Comment:
Perhaps add a note here to clarify this additional test's purpose

Reply:
I agree, done